cubids/cubids.py

```diff
@@ -1734,7 +1766,7 @@ def get_entity_value(path, key):
             return part
 
 
-def build_path(filepath, entities, out_dir):
+def build_path(filepath, entities, out_dir, is_longitudinal=False):
```

Comment:
For internal functions, removing the default value will make it easier to track down errors like we were seeing earlier.